### ISSUE TITLE
Feature Request: Enhance UI with Two Language Drop-Down Lists

### ISSUE THREAD

**Issue by boroli** [XapaJIaMnu/translateLocally#169]
Similar to the website, it would be nice to have two drop-down lists, one for the language to translate from, and once selected the first language, the second Drop-Down List should display all available languages to translate to. Maybe greyed out those for which no translation library was downloaded yet (and have a click icon to start the download easily in the background - if Internet connection is available).
And then additionally having an arrow in the middle, to switch forth and back between those languages. Basically similar to the website UI. The last chosen languages should be stored locally. Thus after restart, it opens with the same languages again.
This would enhance usability greatly, as it reduces the number of clicks one has to do, if changing between languages.

**Comment by XapaJIaMnu** [comment]
Yeah, we want to do that but alas all developers are now out of time to do the necessary modifications.
